packages/react-components/react-color-picker-preview/library/src/components/ColorArea/ColorArea.test.tsx

```diff
@@ -7,12 +7,22 @@ describe('ColorArea', () => {
   isConformant({
     Component: ColorArea,
     displayName: 'ColorArea',
+    disabledTests: ['component-handles-ref', 'component-has-root-ref'],
```

Comment:
Could you please explain the reason for removing the ref from the component?

Reply:
That ref was always null. I added ref as `const ref = React.useRef(null)`

Reply:
Shouldn't we forward ref to the cmponent?

Reply:
Yeah. This will be part of the second PR with keyboard events. I'll use `input` as a primary slot